packages/smooth_app/assets/onboarding/sample_product_data.json

```diff
@@ -276,15 +276,15 @@
             "attributes" : [
                {
                   "description" : "",
-                  "description_short" : "Very low environmental impact",
-                  "grade" : "a",
-                  "icon_url" : "https://static.openfoodfacts.org/images/attributes/dist/ecoscore-a.svg",
+                  "description_short" : "Unknown environmental impact",
+                  "grade" : "unknown",
+                  "icon_url" : "https://static.openfoodfacts.org/images/attributes/dist/ecoscore-unknown.svg",
                   "id" : "ecoscore",
-                  "match" : 100,
+                  "match" : 0,
                   "name" : "Eco-Score",
                   "panel_id" : "ecoscore",
-                  "status" : "known",
-                  "title" : "Eco-Score A"
```

Comment:
Mhhh the example product seems to break all the time

Reply:
can we have something more radical than producer platform protection ? @stephane
Or is it the opportunity to fix partial protection ?